cloud/azure/modules/bastion/main.tf

```diff
@@ -0,0 +1,91 @@
+# Make sure the bastion is within the vnet of the database
+resource "azurerm_subnet" "bastion_subnet" {
+  name                 = "bastion_subnet"
+  resource_group_name  = var.resource_group_name
+  virtual_network_name = var.vnet_name
+  address_prefixes = [
+    "10.0.6.0/24"
```

Comment:
question (blocking) should this be a variable? I think we made the other subnet prefixes variables, so let's keep it consistent.

Reply:
done!